src/marina.html

```diff
@@ -147,13 +146,32 @@ <h1 class="text-center">Marina is the ultimate gateway designed to access the Li
         <div class="column w-col w-col-6 w-col-stack">
           <img src="images/firefox-logo.png" loading="lazy" alt="">
           <div class="space-x-small"></div>
-          <p class="text-center">Firefox (coming soon)</p>
+          <p class="text-center">Firefox</p>
```

Comment:
wrap it already as clickable, we will then update the link only as soon we publish

Reply:
done in 01edd47